### PR TITLE
TLSPE-55: fix update urls

### PR DESCRIPTION
**Pull request by dlhaines**

### CODE REVIEW

File: src/test/java/edu/umich/its/spe/SPEMasterProcessGradeTest.java [hunk 1]
```diff
@@ -48,24 +48,26 @@ public void testConvertSPEGradesFromDataWarehouseJSON() throws JSONException{
 
 		for(HashMap<String, String> e : gradedFromDW) {
 			assertThat("proper type",e,instanceOf(HashMap.class));
-			assertThat("Score exists and is string",e.get("Score"),instanceOf(String.class));
-			assertThat("Unique_Name exists and is string",e.get("Unique_Name"),instanceOf(String.class));
+			assertThat("Score exists and is string",e.get(SPEMaster.SCORE),instanceOf(String.class));
+			assertThat("Unique_Name exists and is string",e.get(SPEMaster.UNIQUE_NAME),instanceOf(String.class));
+//			assertThat("Score exists and is string",e.get("Score"),instanceOf(String.class));
```
Comment (zqian) [inline src/test/java/edu/umich/its/spe/SPEMasterProcessGradeTest.java:53]: remove those two commented lines?

File: src/test/java/edu/umich/its/spe/SPEMasterProcessGradeTest.java [hunk 2]
```diff
@@ -48,24 +48,26 @@ public void testConvertSPEGradesFromDataWarehouseJSON() throws JSONException{
...
 		}
 	}
 
 	@Test
 	public void testConvertAssignmentAToGradeMap() throws JSONException {
 		JSONObject jo = new JSONObject(assignmentJSON_A);
 		HashMap<String, String> gradeMap = SPEMaster.convertAssignmentToGradeMap(jo);
-		assertEquals("score:","1721.5",gradeMap.get("Score"));
+		assertEquals("score:","1721.5",gradeMap.get(SPEMaster.SCORE));
 		assertEquals("uniqname:","studentc",gradeMap.get("Unique_Name"));
```
Comment (zqian) [inline src/test/java/edu/umich/its/spe/SPEMasterProcessGradeTest.java:63]: replace grade.getString("Unique_Name") with grade.getString(SPEMaster.UNIQUE_NAME)?

File: src/test/java/edu/umich/its/spe/SPEMasterProcessGradeTest.java [hunk 3]
```diff
@@ -75,7 +77,7 @@ public void testConvertAssignmentBToGradeMap() throws JSONException {
 	public void testConvertAssignmentCToGradeMap() throws JSONException {
 		JSONObject jo = new JSONObject(assignmentJSON_C);
 		HashMap<String, String> gradeMap = SPEMaster.convertAssignmentToGradeMap(jo);
-		String score = gradeMap.get("Score");
+		String score = gradeMap.get(SPEMaster.SCORE);
 		assertTrue("allow score with >4 numbers before decimal: ["+score+"]",SPEMaster.scoreRegexPattern.matcher(score).matches());
 		assertEquals("score:","945.5",score);
 		assertEquals("uniqname:","studentd",gradeMap.get("Unique_Name"));
```
Comment (zqian) [inline src/test/java/edu/umich/its/spe/SPEMasterProcessGradeTest.java:83]: replace grade.getString("Unique_Name") with grade.getString(SPEMaster.UNIQUE_NAME)?

File: src/test/java/edu/umich/its/spe/SPEMasterProcessGradeTest.java [hunk 4]
```diff
@@ -93,7 +95,7 @@ public void testJSONObjectParse() throws JSONException {
 
 			JSONObject grade = jAD.getJSONObject(i);
 			assertThat("proper type",grade,instanceOf(JSONObject.class));
-			assertThat("Score exists and is string",JSONObject.valueToString(grade.get("Score")),instanceOf(String.class));
+			assertThat("Score exists and is string",JSONObject.valueToString(grade.get(SPEMaster.SCORE)),instanceOf(String.class));
 			assertThat("Unique_Name exists and is string",grade.getString("Unique_Name"),instanceOf(String.class));
```
Comment (zqian) [inline src/test/java/edu/umich/its/spe/SPEMasterProcessGradeTest.java:99]: replace grade.getString("Unique_Name") with grade.getString(SPEMaster.UNIQUE_NAME)?

**Comment by dlhaines** [comment]
Good comments.  Will implement in future if more changes are made.

**Comment by dlhaines** [comment]
All good comments.

**Pull request merged**